### ISSUE TITLE
Template creation/update not working in some cases

### ISSUE THREAD

**Issue by mazondo** [sendwithus/grunt-sendwithus#1]
Some templates are failing to update, and others are failing to be created, but doing so silently so it looks like the change was successful.

I added logging output to determine what the actual response was:

plugin output:

```
Uploaded dist/template-name to sendwithus
```

actual API response:

```
response: Template not found
```

In other cases:

plugin output:

```
Uploaded dist/owner-booking-day-before-dropoff.html to sendwithus
```

api response:

```
response: Invalid email subject
```

and in other cases:

plugin output:

```
└── Updated version undefined in API
```

api response:

```
response: ValidationError: expected token 'end of print statement', got 'string' on line 185
```

**Comment by mazondo** [comment]
Added additional logging.  request url is being set as `https://api.sendwithus.com/api/v1/templates/undefined/versions`

**Comment by brandonb927** [comment]
Hey @mazondo, thanks for bringing this to our attention! I'll take a look at this as soon as I can and let you know if anything transpires.

🍻

**Comment by brandonb927** [comment]
@mazondo We just published version `0.1.0` to npm, can you try downloading it and testing it out please? :)
